src/shared/components/Dialog/Dialog.style.ts

```diff
@@ -0,0 +1,124 @@
+import styled from '@emotion/styled'
+import { css } from '@emotion/react'
+import { colors, sizes, breakpoints, typography, zIndex } from '../../theme'
+import { Text, Button } from '@/shared/components'
+import { SIDENAVBAR_WIDTH } from '@/components/SideNavbar/SideNavbar.style'
+
+type StyledExitButtonProps = {
+  marginLeft?: boolean
+}
+
+export const StyledBackdrop = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  right: 0;
+  height: 100vh;
+  background-color: rgba(0, 0, 0, 0.4);
+  z-index: ${zIndex.globalOverlay};
+  @media (min-width: ${breakpoints.medium}) {
+    padding-left: ${SIDENAVBAR_WIDTH}px;
+  }
+`
+
+export const StyledContainer = styled.div`
+  width: 90%;
+  max-width: ${sizes(110)};
+  margin: ${sizes(32)} auto;
+  background-color: ${colors.gray[600]};
+  padding: ${sizes(4)};
+  box-shadow: 0px 8px 8px rgba(0, 0, 0, 0.12), 0px 24px 40px rgba(0, 0, 0, 0.16);
+  @media screen and (min-width: ${breakpoints.small}) {
+    padding: ${sizes(6)};
+  }
+`
+
+export const StyledTitleText = styled(Text)`
+  margin-bottom: ${sizes(3)};
+`
+
+export const StyledContentText = styled(Text)`
+  color: ${colors.white};
+  font-weight: ${typography.weights.thin};
+  margin-bottom: ${sizes(6)};
+  word-wrap: break-word;
```

Comment:
```suggestion
  word-wrap: break-word;
  line-height: ${sizes(5)};
```
Content and Title definitely needs some air here